package.json

```diff
@@ -1,15 +1,38 @@
 {
-    "name": "root",
-    "private": true,
+    "name": "@nypl/design-system-react-components",
+    "version": "0.4.0",
+    "description": "Design System React Components",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/NYPL/nypl-design-system/"
+    },
+    "main": "dist/design-system-react-components.min.js",
+    "types": "lib/index.d.ts",
+    "author": "NYPL",
+    "license": "Apache-2.0",
     "scripts": {
-        "prepare": "lerna run prepare",
-        "install-all": "npm install; lerna run prepare; lerna exec npm install",
-        "start": "lerna exec --scope storybook npm run start:react",
-        "storybook": "lerna exec --scope storybook npm run start:react",
-        "storybook:static": "lerna exec --scope storybook npm run static:react",
-        "test": "lerna exec npm test",
+        "prepare": "npm run build",
+        "start": "node node_modules/@storybook/react/bin/index -p 9001 -c .storybook-react",
+        "static": "node node_modules/.bin/build-storybook -c .storybook-react -o ./storybook-static/react",
+        "test": "rm -rf lib && npm run lint && tsc && cp src/styles.scss lib && mocha --require lib/__tests__/setupEnzyme.js --recursive lib/**/*.test.js",
         "format": "prettier --write 'src/**/*.{ts, tsx, js,jsx,json}'",
-        "pretty-quick": "pretty-quick"
+        "pretty-quick": "pretty-quick",
+        "prepublishOnly": "npm run test && npm run dist",
+        "build": "rm -rf lib && tsc",
+        "dist": "rm -rf dist && webpack --mode production --progress --display-modules --config webpack.config.js",
```

Comment:
What is the difference between `npm run build`, `dist` and `prepare`?

Reply:
It looks like running `npm link` runs `dist`, which builds into `/lib`, but the package.json lists main as `dist/design-system-react-components.min.js`

Reply:
Thanks for bringing this up, I agree this should be more clear.  

`build` is building the ts files into `lib`.  The tests are running from `lib`. `dist` is packaging everything for publish.  I'm not quite sure why the intermediate `lib` step needs to happen though - when I set it up I think I was just happy to have it working, so I'm happy to rethink it. 
 
`npm link` should be run with `npm run watch-webpack` for local development, I will update the readme to reflect this.

`prepare` isn't doing anything - really `build` should be run in `prepublishOnly` so I'll have it reflect that

Reply:
Ah I see. There should be a way to make Mocha parse the typescript I assume if you want to update that. But it's not the end of the world either way